apps/bfd-server/bfd-server-launcher/src/test/java/gov/cms/bfd/server/launcher/DataServerLauncherAppIT.java

```diff
@@ -29,6 +33,32 @@
 public final class DataServerLauncherAppIT {
   /** The POSIX signal number for the <code>SIGTERM</code> signal. */
   private static final int SIGTERM = 15;
+  /** Regex for access log entries */
+  private static final String accessLogPattern =
+      "^(\\S+) \\S+ \\\"([^\\\"]*)\\\" \\[([^\\]]+)\\] \\\"([A-Z]+) ([^ \\\"]+) HTTP\\/[0-9.]+\\\" \\\"([^ \\\"]+)\\\" ([0-9]{3}) ([0-9]+|-) ([0-9]+|-) (\\S+) ([0-9]+|-) \\[([^\\]]+)\\] ([0-9]+|-) \\\"([^\\\"]*)\\\" ([0-9]+|-) \\\"([^\\\"]*)\\\" ([0-9]+|-) \\\"([^\\\"]*)\\\" (\\S+)";
+
+  /**
+   * Verifies the regex for valdiating our access log entries adequately avoids edge cases that
+   * could break our alerts, which depend on logs
+   */
+  @Test
+  public void checkAccessLogFormat() {
+    String goodLine1 =
+        "127.0.0.1 - \"CN=client-local-dev\" [07/Mar/2022:18:43:15 +0000] \"GET / HTTP/1.1\" \"?null\" 200 26 22000 - - [-] - \"-\" - \"-\" - \"-\" -";
+    String goodline2 =
+        "10.252.14.216 - \"EMAILADDRESS=gomer.pyle@adhocteam.us, CN=BlueButton Root CA, OU=BlueButton on FHIR API Root CA, O=Centers for Medicare and Medicaid Services, L=Baltimore, ST=Maryland, C=US\" [01/Oct/2021:23:10:01 -0400] \"GET /v1/fhir/Coverage/?startIndex=0&_count=10&_format=application%2Fjson%2Bfhir&beneficiary=Patient%2F587940319 HTTP/1.1\" \"?startIndex=0&_count=10&_format=application%2Fjson%2Bfhir&beneficiary=Patient%2F587940319\" 200 2103 23 3b3e2b30-232f-11ec-9b9f-0a006c0cb407 1 [2021-10-02 03:10:01.104125] 11770 \"-\" 32 \"Evidation on behalf of Heartline\" 79696 \"-\" patientId:587940319";
```

Comment:
looks like they were a couple of lines pulled from a real access.log which is nice -- can you document that in a comment?